### PR TITLE
website(showcase): add new projects (Expo, Currenxie).

### PR DESCRIPTION
**Pull request by asafkorem**
The PR adds two new showcased projects (Expo, Currenxie) and replaces the fixed arbitrary order of apps with a shuffled order on every load.

### CODE REVIEW

File: website/src/components/Showcase/Showcase.js [hunk 1]
```diff
@@ -5,8 +5,19 @@ import cardList from '@site/showcase.json';
...
 
 function Showcase() {
-  const cards = cardList.map((props) => <Card key={props.title} {...props} />);
+  const cards = _makeShuffledCardList().map(_makeCard)
+  return _makeCardsContainer(cards);
+}
+
+const _makeShuffledCardList = () => {
+  return cardList.sort(() => 0.5 - Math.random());
+}
```
Comment (d4vidi) [inline website/src/components/Showcase/Showcase.js:14]:
```suggestion
const _makeShuffledCardList = () => (cardList.sort(() => 0.5 - Math.random()));
```

File: website/src/components/Showcase/Showcase.js [hunk 2]
```diff
@@ -5,8 +5,19 @@ import cardList from '@site/showcase.json';
...
+
+const _makeCard = (props) => {
+  return <Card key={props.title} {...props} />;
+}
```
Comment (d4vidi) [inline website/src/components/Showcase/Showcase.js:18]:
```suggestion
const _makeCard = (props) => <Card key={props.title} {...props} />;
```

File: website/src/components/Showcase/Showcase.js (hunk 1 above)
Comment (d4vidi) [reply website/src/components/Showcase/Showcase.js:14]: I think we should keep the Wix ones at the top:
1. Would be weird to have it somewhere in the middle
2. Need to stay adjacent, as a group

Comment (asafkorem) [reply website/src/components/Showcase/Showcase.js]: OK, I agree on both points.. Done

File: website/src/components/Showcase/Showcase.js (hunk 2 above)
Comment (asafkorem) [reply website/src/components/Showcase/Showcase.js]: Done :+1:

File: website/src/components/Showcase/Showcase.js [hunk 3]
```diff
@@ -10,19 +10,18 @@ function Showcase() {
 }
 
 const _makeShuffledCardList = () => {
-  return cardList.sort(() => 0.5 - Math.random());
+  const alwaysOnTop = cardList.filter(card => card.shouldStayOnTop);
+  const shuffled = cardList.filter(card => !card.shouldStayOnTop).sort(() => Math.random() - 0.5);
+  return alwaysOnTop.concat(shuffled);
```
Comment (d4vidi) [inline website/src/components/Showcase/Showcase.js:15]: [Lodash partition[(https://lodash.com/docs/4.17.15#partition) 🤔

Comment (asafkorem) [reply website/src/components/Showcase/Showcase.js:15]: I thought about that but didn't care enough 😄

Comment (asafkorem) [reply website/src/components/Showcase/Showcase.js:15]: anyway https://github.com/wix/Detox/pull/3696

**Pull request merged**